packages/eslint-plugin-udemy/rules/README_TEMPLATE.md

```diff
@@ -0,0 +1,23 @@
+# < Capitalized Name of the Rule >
+
+< A sentence or two describing the details of what this rule is trying to achieve. >
+
+## Rule Details
+
+< Explanation of any existing arguments for the rule. Or a statement lack thereof. >
```

Comment:
Explain any arguments that the rule accepts or explain that it does not accept any arguments.

Reply:
I was really hoping to use "lack thereof" :D

Reply:
...or a statement stating your lack thereof.